src/numerics/numeric_vector.C

```diff
@@ -430,6 +431,29 @@ bool NumericVector<T>::compatible (const NumericVector<T> & v) const
 }
 
 
+template <typename T>
+bool NumericVector<T>::fuzzy_equal(const NumericVector<T> & v, const Real tol) const
+{
+  bool equiv = true;
+  if (this->local_size() != v.local_size())
+    equiv = false;
+
+  if (equiv)
+    for (const auto i : make_range(this->first_local_index(), this->last_local_index()))
+    {
+      if (relative_fuzzy_equal((*this)(i), v(i), tol) || absolute_fuzzy_equal((*this)(i), v(i), tol))
```

Comment:
I don't really agree. I've had multiple times where I've checked to see whether a vector/matrix is equivalent purely with a relative equal check only to be disappointed when I got `false` because of `1e-15` vs `3e-15`. I suspect the users of these methods will really only care for this to return `false` if two matrices are "really" different.

Reply:
I would be game for passing in two different tolerances though, or one relative tolerance and a floor

Reply:
I do like the two different tolerances idea, just pass in 0 for the one you don't want.

But honestly, there are places to use a relative equal check and there are places to use an absolute equal check, and I'm having trouble imagining where you'd want to use both.  If you know the scaling of your problem then passing that in as an absolute tolerance is the right way to go, and if you don't know the scaling of your problem, `1e-15` vs `3e-15` theoretically could be a significant issue, and with our default tolerances `1e-7` vs `3e-7` actually has been a significant issue.